### PR TITLE
add migrate va to 0.8 doc

### PR DESCRIPTION
**Pull request by lzc850612**
### Purpose
*What is the context of this pull request? Why is it being done?*

In 0.8 release we introduce skill capabilities with BotBuilder and this PR adds documentation to explain how to migrate old Virtual Assistant to use the new capabilities 0.8 has to offer

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.7.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.7.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.7.2-preview" />
+    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.7.2" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.7.2" />
+    <PackageReference Include="Microsoft.Bot.Schema" Version="4.7.2" />
+
+```
+
+5. Change the Microsoft.Bot.Builder.Solutions package to Microsoft.Bot.Solutions with the version 0.8.0-preview. If your Virtual Assistant project has a reference to the package Microsoft.Bot.Builder.Skills, please remove it
```
Comment (pavolum) [inline docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md]: Might want to add that they will have to change all corresponding references although Im sure thatll be obvious once they build and see all the errors.

**Review by pavolum — APPROVED** [review]

File: docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md (hunk 1 above)
Comment (lzc850612) [reply docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md]: Yeah all corresponding references would be all the botbuilder ones talked about in #4.

Comment (pavolum) [reply docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md]: I meant the import statements in the individual files throughout the solution. Their old VA will still have 'using Microsoft.Bot.Builder.Solutions' which wont exist after the refresh nuget packages.

Comment (lzc850612) [reply docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md]: shoot yes somehow i forgot about that. let me add one item for that.

**Comment by lzc850612** [comment]
won't merge this until after the release

File: docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md (hunk 1 above)
Comment (darrenj) [reply docs/_docs/overview/whats-new/0.8-beta/migrate-old-va-to-0.8.md]: Addressed

**Review by darrenj — APPROVED** [review]

**Comment by darrenj** [comment]
Ready to merge for the release

**Pull request merged**